index.js

```diff
@@ -1,2 +1,94 @@
+// Load environment variables
+require('dotenv').config();
+
+const slackEventsApi = require('@slack/events-api');
+const passport = require('passport');
+const SlackStrategy = require('@aoberoi/passport-slack').default.Strategy;
+const http = require('http');
 const express = require('express');
-const slack = require('slack');
\ No newline at end of file
+const bodyParser = require('body-parser');
+
+const { getClientByTeamId } = require('./util/getClient');
+
+const port = process.env.APP_PORT || 3000;
+
+// Initialize a data structures to store team authorization info (typically stored in a database)
```

Comment:
rm comment imo, well written code wont need comments (lets introduce comments once our app gets complex, not when its <100 lines of code)